app/models/booking_response.rb

```diff
@@ -31,6 +31,7 @@ class BookingResponse
     :prisoner_full_name, :prisoner_number, :prisoner_date_of_birth,
     :contact_email_address, :contact_phone_no,
     :visitors,
+    :processable?, :human_processing_state_name,
```

Comment:
where does `human_processing_state_name` come from in `Visit`? Anyway, I think this is misnamed: it's not human, but internal, and we only convert to human at the i18n layer.